### PR TITLE
release 1.0.0

### PR DESCRIPTION
**Pull request by awalterschulze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,9 @@ erlfmt is an opinionated Erlang code formatter. By automating the process of
 formatting the code, it frees your team up to focus on what the code does,
 instead of what it looks like.
 
-erlfmt is feature complete and now is a final opportunity,
-before the release of version 1.0.0, to influence formatting decisions.
+erlfmt is feature complete and released as version 1.0
+This means only backwards incompatible changes and bug fixes can be adopted without very serious consideration.
```
Comment (gomoripeti) [inline README.md:8]: nitpick: only backwards *compatible*
(btw 🎉 )

Comment (awalterschulze) [reply README.md]: Great catch

**Review by michalmuskala — APPROVED** [review]
Looks great. This is an awesome milestone 🎉

**Comment by awalterschulze** [comment]
> Looks great. This is an awesome milestone 🎉

We did it together  🎉

**Pull request merged**